docker-compose.yml

```
@@ -1,13 +1,25 @@
 version: "3"
 services:
-  starknet-node:
+  starknet-goerli:
     image: eqlabs/pathfinder:latest
-    restart: unless-stopped
     environment:
       - RUST_LOG=info
-    env_file:
-      - pathfinder-var.env
+      - PATHFINDER_ETHEREUM_API_URL=${PATHFINDER_ETHEREUM_API_URL_GOERLI}
+      - PATHFINDER_HTTP_RPC_ADDRESS=0.0.0.0:9545
     volumes:
-      - ./pathfinder:/usr/share/pathfinder/data
+      - goerli-data:/usr/share/pathfinder/data
     ports:
       - 9545:9545
+  starknet-mainnet:
+    image: eqlabs/pathfinder:latest
+    environment:
+      - RUST_LOG=info
+      - PATHFINDER_ETHEREUM_API_URL=${PATHFINDER_ETHEREUM_API_URL_MAINNET}
+      - PATHFINDER_HTTP_RPC_ADDRESS=0.0.0.0:9546
```

This should be the same as above:

```suggestion
      - PATHFINDER_HTTP_RPC_ADDRESS=0.0.0.0:9545
```

Containers run in separate network namespaces so binding both to the same port should be OK.

---

If I put the same ports, even locally I have

```
docker compose up         
[+] Running 2/2
 ⠿ Container pathfinder-starknet-goerli-1   Created                                                                                                                                     0.0s
 ⠿ Container pathfinder-starknet-mainnet-1  Recreated                                                                                                                                   0.2s
Attaching to pathfinder-starknet-goerli-1, pathfinder-starknet-mainnet-1
Error response from daemon: Ports are not available: exposing port TCP 0.0.0.0:9545 -> 0.0.0.0:0: listen tcp 0.0.0.0:9545: bind: address already in use
```

---

It's not compatible with AWS integration (you can run the AWS snippet to get this error)

```
published port can't be set to a distinct value than container port: incompatible attribute
```